packages/picasso-lab/src/TextEditor/hooks/useAutofocus/test.ts

```diff
@@ -0,0 +1,33 @@
+import { renderHook } from '@testing-library/react-hooks'
+
+import useAutofocus from './useAutofocus'
+
+describe('useAutofocus', () => {
+  it('does not focus quill', () => {
```

Comment:
```suggestion
  it('does not focus on editor when autofocus is off', () => {
```

Reply:
I would wrap with `describe` as well:

```suggestion
  describe('when autofocus is off', {
    it('does not focus on editor', () => {
```

Reply:
I like dmitrys suggestion more, so I wrapped it in another describe.